templates/server/src/main/resources/archetype-resources/pom.xml

```diff
@@ -79,6 +84,7 @@
 
       <plugin>
         <artifactId>maven-surefire-plugin</artifactId>
+        <version>2.22.0</version>
         <configuration>
           <excludedGroups>${devonfw.test.excluded.groups}</excludedGroups>
```

Comment:
The categories have changed to tags with different IDs. I am missing the diff to update these:
https://github.com/AbhayChandel/devon4j/blob/issue-130-update-JUnit-to-5.x/templates/server/src/main/resources/archetype-resources/pom.xml#L18

Does that still work with `excludedGroups` attributes after switching to tags?